OrbitBase/include/OrbitBase/internal/identity.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_INTERNAL_IDENTITY_H_
+#define ORBIT_BASE_INTERNAL_IDENTITY_H_
+
+namespace internal {
+
+template <typename T>
+struct identity {
```

Comment:
Can we follow this as closely as possible naming- and implementation-wise? https://en.cppreference.com/w/cpp/types/type_identity